packages/evm/src/fungible.ts

```diff
@@ -0,0 +1,111 @@
+import type {
+  Domain,
+  EvmResource,
+  FeeHandlerType,
+} from "@buildwithsygma/sygma-sdk-core";
+import type { SecurityModel } from "../../core/src/index.js";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+interface TransactionRequest {
+  to: string;
+  value: bigint;
```

Comment:
why is this needed?
the sdk can directly return `PopulatedTransaction` object and arrays

Reply:
Isn't this `PopulatedTransaction` ethers specific type @saadjhk? If it is, then the idea is not to return lib. specific types from the SDK. cc: @mpetrunic, correct me if I misunderstood the idea.

Reply:
yup,  idea is to have ability to eventually remove ethers from the lib and replace it with something smaller/less bloated

Reply:
@mpetrun5 contract interfaces and package is heavily dependent on ethers as well

Reply:
True, but thats internal and it's easy to replace. Both web3.js and viem doesn't require typechain for types anymore and can consume json abi directly